data/desktop/io.snapcraft.SessionAgent.desktop.in

```diff
@@ -0,0 +1,7 @@
+[Desktop Entry]
+Type=Application
+Name=Snapd User Session Agent
```

Comment:
This probably needs to be translatable but I have long forgotten how to handle localized desktop files.

Reply:
It doesn't even look like we currently install the message catalogues in the source tree, and they haven't been updated since November 2019.  I agree that we should be extracting strings from the desktop files though, and possibly merging translations back in.  This does become a bit more important as snapd starts adding regular user facing UI.